src/GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -1358,6 +1358,11 @@ public bool IsExistingInTfs(string path)
             return VersionControl.ServerItemExists(path, VersionSpec.Latest, DeletedState.Any, ItemType.Any);
         }
 
+        public bool ValidateTfsFolder(string remoteTfsPath)
+        {
+            return VersionControl.ServerItemExists(remoteTfsPath, VersionSpec.Latest, DeletedState.NonDeleted, ItemType.Folder);
```

Comment:
`DeletedState.NonDeleted` means it will fail for any folder which is deleted in the meantime, meaning you can't clone the history of a folder which gets deleted

Reply:
I saw that, but I was thinking they would not want to clone deleted folders.

Reply:
A valid use case would be to convert some history TFS history to git. 
Although I admit it would be rare use case, but still worthwhile to support.